uber/templates/registration/discount.html

```diff
@@ -8,6 +8,9 @@
 {% csrf_token %}
 <table class="form">
 <tr>
+    <!-- Note that this is preferred first and last name.
+    May want to change, probably by sending an attendee ID to this page as a hidden field/something,
+    instead of having to re-search for them with potentially ambiguous/problematic results. -->
```

Comment:
This page is an admin form which creates a link we then give to an attendee, at which point they can fill out the rest of their information when they complete the registration and pay.  I think we can therefore just leave this form as-is, since all of the personal information can be edited by the attendee later.